src/gui/mzroll/downloadmanager.cpp

```diff
@@ -0,0 +1,89 @@
+#include "downloadmanager.h"
+#include <QNetworkAccessManager>
+#include <QNetworkRequest>
+#include  <QNetworkReply>
+#include <QDebug>
+#include "pollyintegration.h"
```

Comment:
@rish9511 Is "pollyintegration.h" being used in downloadmanager? If not, let's remove this dependency

Reply:
@shubhra-agrawal  
Yes, we are using it.

Reply:
Yes, it's being used

https://github.com/ElucidataInc/ElMaven/pull/1055/files#diff-de0fde6efde85e103aaf21acf14c9f3aR20